### DISCUSSION TITLE
What in the world is this repo?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#21] [General]
Why did you name this repo, JackSucksAtLife's biggest channel name due to the YT glitch lol? https://github.com/TechStudent11/JACKSEPICYOUTUBECHANNELFULLOFFUNTIMESANDFUNHIRICKXHASNOTHINGONMEIAMTHESUPERIORCHANNELIHAVEMORECHARAC the original project is called **alphabet**

**Comment by TechStudent10** [comment]
Ok so I was afraid that you would find some embarrassing repo from like a year ago haha. Mttrms/alphabet is a game where you have to type the alphabet as fast as you can. I forked it and replaced with JACKSEPICYOUTUBECHANNELFULLOFFUNTIMESANDFUNHIRICKXHASNOTHINGONME...

I didn't you watched JackSucksAtLife haha.

**Reply by KendallDoesCoding** [reply]
Ok hahaha

I watch him sometimes, yes
